runtime/src/migration.rs

```diff
@@ -1,28 +1,24 @@
 use crate::VERSION;
-use sr_primitives::print;
+use sr_primitives::{print, traits::Zero};
 use srml_support::{decl_event, decl_module, decl_storage};
 use sudo;
 use system;
 
-// When preparing a new major runtime release version bump this value to match it and update
-// the initialization code in runtime_initialization(). Because of the way substrate runs runtime code
-// the runtime doesn't need to maintain any logic for old migrations. All knowledge about state of the chain and runtime
-// prior to the new runtime taking over is implicit in the migration code implementation. If assumptions are incorrect
-// behaviour is undefined.
-const MIGRATION_FOR_SPEC_VERSION: u32 = 0;
-
 impl<T: Trait> Module<T> {
-    fn runtime_initialization() {
-        if VERSION.spec_version != MIGRATION_FOR_SPEC_VERSION {
-            return;
-        }
-
-        print("running runtime initializers");
+    fn runtime_upgraded() {
+        print("running runtime initializers...");
 
         // ...
-        // add initialization of other modules introduced in this runtime
+        // add initialization of modules introduced in new runtime release. This
+        // would be any new storage values that need an initial value which would not
+        // have been initialized with config() or build() mechanism.
         // ...
 
+        // Create the Council mint. If it fails, we can't do anything about it here.
+        let _ = governance::council::Module::<T>::create_new_council_mint(
```

Comment:
We agreed to print an error message in such cases. Don't we?

Reply:
Done in https://github.com/Joystream/substrate-runtime-joystream/pull/275/commits/a6148b9a91b7c179ec0eb729ff1584e36b59bddb